src/components/PupilViews/PupilUnitListing/PupilUnitListing.view.tsx

```diff
@@ -13,20 +13,26 @@ import { UseBackHrefProps, useBackHref } from "./useBackHref";
 import { UnitsSectionData } from "@/pages/pupils/programmes/[programmeSlug]/units";
 import SignpostTeachersInlineBanner from "@/components/PupilComponents/SignpostTeachersInlineBanner/SignpostTeachersInlineBanner";
 import { PupilUnitsSection } from "@/components/PupilComponents/PupilUnitsSection";
+import useAnalytics from "@/context/Analytics/useAnalytics";
+import { UnitListingBrowseData } from "@/node-lib/curriculum-api-2023/queries/pupilUnitListing/pupilUnitListing.schema";
+import { generateKeyStageTitle } from "@/components/PupilComponents/PupilAnalyticsProvider/PupilAnalyticsProvider";
 
 export type PupilViewsUnitListingProps = {
   unitSections: UnitsSectionData[];
   phase: "primary" | "secondary";
   backHrefSlugs: UseBackHrefProps;
   subjectCategories: string[];
+  programmeFields: UnitListingBrowseData[number]["programmeFields"];
 };
 
 export const PupilViewsUnitListing = ({
   unitSections,
   phase,
   backHrefSlugs,
   subjectCategories,
+  programmeFields,
```

Comment:
We already have the programme fields inside unitSections so this is a bit redundant. I'd say leave this one for now though.